### PR TITLE
Javascript/javascript3/week3

### PR DESCRIPTION
**Pull request by JyothiNandyala**
added homework for javascript3 week3.

### CODE REVIEW

File: javascript/javascript3/week3/circle.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Canvas Circle</title>
+</head>
+<body>
+    <canvas id="mycanvas"></canvas>
```
Comment (qamarfj) [inline javascript/javascript3/week3/circle.html:9]: you can write id like my-canvas.

**Review by qamarfj — COMMENTED** [review]
Nice work.

File: javascript/javascript3/week3/circle.html (hunk 1 above)
Comment (JyothiNandyala) [reply javascript/javascript3/week3/circle.html:9]: ok thank you

File: javascript/javascript3/week3/circle.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        context.stroke();
+        context.fillStyle = this.color;
+        context.fill();
+    }
+}
+let createCircle = function () {
+    let randox_x = Math.random() * window.innerWidth;
+    let randox_y = Math.random() * window.innerHeight;
+    let random_radius = Math.floor(Math.random() * 100);
+    let random_color = 'rgb(' + Math.floor(Math.random() * 255) + ',' + Math.floor(Math.random() * 255) + ',' + Math.floor(Math.random() * 255) + ')';
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/circle.js:28]: `Math.floor(Math.random() * 255)` is used 3 times. Better to extract it into a separate function, and this code will be shorter and less error-prone

File: javascript/javascript3/week3/circle.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    circleObject.draw()
+}
+setInterval(createCircle, 100)
+
+//to draw the circles with cursor move
+window.addEventListener('mousemove', function (event) {
+    let randox_x = event.x;
+    let randox_y = event.y;
+    let random_radius = Math.floor(Math.random() * 100);
+    let random_color = 'rgb(' + Math.floor(Math.random() * 255) + ',' + Math.floor(Math.random() * 255) + ',' + Math.floor(Math.random() * 255) + ')';
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/circle.js:41]: same story here.

File: javascript/javascript3/week3/githubrepos.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    .then(files => {
+        files.forEach(file => {
+            renderPromises(file.json())
+        })
+    })
+    .catch(err => {
+        console.log(err)
+    })
+
+let renderPromises = (prom) => {
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/githubrepos.js:14]: `let` could be turned into a `const`. Function definitions are almost never changed.

File: javascript/javascript3/week3/shoppingCart.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
+class Product {
+    constructor(name, price) {
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/shoppingCart.js:2]: `priceInDKK` might be a better name

**Review by aleksd-dfds — APPROVED** [review]

**Pull request merged**